sdks/python/apache_beam/examples/complete/estimate_pi.py

```diff
@@ -106,6 +106,9 @@ def run(argv=None):
                       required=True,
                       help='Output file to write results to.')
   known_args, pipeline_args = parser.parse_known_args(argv)
+  # We use the save_main_session option because one or more DoFn's in this
+  # workflow rely on global context (e.g., a module imported at module level).
+  pipeline_args.append('--save-main-session')
```

Comment:
I'd rather encourage the patter of creating an PipelineOptions and programatically setting this value rather than manipulating the flat strings.